ietf-layer0-types.yang

```diff
@@ -1171,73 +1214,22 @@ module ietf-layer0-types {
     }
   }
 
-  grouping cd-pmd-penalty {
-    description "entries of table; triplet chromatic
-    dispersion, polarization mode dispersion and
-    associated penalty";
-
-    leaf chromatic-dispersion {
-      type union {
-        type decimal64 {
-          fraction-digits 2;
-          range "0..max";
-        }
-        type empty;
-      }
-      units "ps/nm";
-      config false;
-      mandatory true;
-      description "chromatic dispersion";
-    }
-    leaf polarization-mode-dispersion {
-      type union {
-        type decimal64 {
-          fraction-digits 2;
-          range "0..max";
-        }
-        type empty;
-      }
-      units "ps";
-      config false;
-      mandatory true;
-      description "Polarization mode dispersion";
-    }
-    leaf penalty {
-      type union {
-        type decimal64 {
-          fraction-digits 2;
-          range "0..max";
-        }
-        type empty;
-      }
-      units "dB";
-      config false;
-      mandatory true;
-      description "Associated penalty on the receiver";
-    }
-  }  
-
-  grouping pdl-penalty {
+  grouping penalty-value {
     description
-      "entries of table; pair of values polarization dependent loss
-      and associated penalty";
+      "A common definition of the penalty value used for describing
+      multiple penalty types (.e.g, CD, PMD, PDL).";
 
-    leaf max-polarization-dependent-loss {
-      type power-in-db-or-null;
-      config false;
-      mandatory true;
-      description
-        "Maximum acceptable accumulate polarization dependent loss";
-    }
-    leaf penalty {
+    leaf penalty-value {
       type union {
         type uint8;
```

Comment:
Shouldn't this be a decimal64 value ? not an int

Reply:
resolving the conversation makes it disappears, but this issue is not solve, I prefer to reopen, not to forget

Reply:
ok